plugins/src/android/OptimoveInitProvider.java

```diff
@@ -23,6 +28,8 @@ public class OptimoveInitProvider extends ContentProvider {
     private static final String IN_APP_AUTO_ENROLL = "auto-enroll";
     private static final String IN_APP_EXPLICIT_BY_USER = "explicit-by-user";
 
+    private static final String ENABLE_DEFERRED_DEEP_LINKING = "enableDeferredDeepLinking";
```

Comment:
Just realising the resources seem to be shared, whilst theoretically unlikely to have key collisions, is it worth prefixing keys in resources?

Reply:
k, updated